src/mscorlib/shared/System/Globalization/DateTimeFormat.cs

```diff
@@ -852,9 +853,15 @@ private static void FormatCustomizedRoundripTimeZone(DateTime dateTime, TimeSpan
                 offset = offset.Negate();
             }
 
-            AppendNumber(result, offset.Hours, 2);
+            Append2DigitNumber(result, offset.Hours);
             result.Append(':');
-            AppendNumber(result, offset.Minutes, 2);
+            Append2DigitNumber(result, offset.Minutes);
+        }
+
+        private static void Append2DigitNumber(StringBuilder result, int val)
+        {
+            result.Append((char)('0' + (val / 10)));
+            result.Append((char)('0' + (val % 10)));
```

Comment:
Could drop the the two `idiv`s?
```csharp
int tens = (int)((val * 205u) >> 11); // div10, valid to 1028
result.Append((char)('0' + tens);
result.Append((char)('0' + (val - (tens * 10));
```

Reply:
Actually the Jit will already do this since its a const

Reply:
> Actually the Jit will already do this since its a const

Yeah, all of the ones that have been commented on here involve a const denominator.  I'm going to let the JIT do its thing.